.github/workflows/pr_check.yml

```diff
@@ -148,13 +148,6 @@ jobs:
           node-version: ${{ env.node-version }}
           cache: 'yarn'
 
-      - name: Setup Java 11
```

Comment:
Did you delete this config to use the default java version specified by the runner?

Reply:
Yep. Default is Java 20